data/web/js/build/011-api.js

```diff
@@ -155,6 +155,8 @@ $(document).ready(function() {
         }
       });
       if (!invalid) {
+        console.log($(this).closest("form").serializeObject());
+        console.log(JSON.stringify($(this).closest("form").serializeObject(), null, 2));

```

Comment:
Are these console.log debug traces?
If yes, they should be remove

Reply:
Same on me 😅